### DISCUSSION TITLE
Support for token exchange flow

### DISCUSSION THREAD

**Discussion by hernangonzalez** [zitadel/zitadel#4088] [General]
Hi!

Project looks good, congrats on making v2!

I have found here that work is supposed to be in progress for the token exchange grant type:
https://docs.zitadel.com/docs/apis/openidoauth/grant-types

Yet I could not see it on the roadmap or issues. Is it accurate?

Many thanks,
H.

**Comment by hifabienne** [comment]
Hi @hernangonzalez 

Sorry, the documentation is wrong at this place.
Actually, we are not working on this right now.

**Reply by hernangonzalez** [reply]
Thanks for letting me know 

Can I request the feature then? 😄

**Reply by hifabienne** [reply]
We have currently a lot of things on our todo list (https://github.com/orgs/zitadel/projects/1). We prioritize on the request of our customers/community.
If we see that a lot of users would love to see this feature we will prioritize it higher.

We also accept PRs if you like to contribute. 😁
